Program.cs

```diff
@@ -952,6 +952,9 @@ static int Main(string[] args)
           return 1;
         }
 
+        // This is a hack, see the comment in the file 
+        engine.Load("ApplicationName.qml");

```

Comment:
```suggestion
        QCoreApplication.OrganizationDomain = "Hammer.mc";
        QCoreApplication.OrganizationName = "Hammer";
```

This seems to work fine (see https://github.com/qmlnet/qmlnet/issues/133)

Reply:
Awesome, much better!